Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -85,6 +91,9 @@ final class ProjectGenerator: ProjectGenerating {
         let sourceRootPath = sourceRootPath ?? project.path
         let xcodeprojPath = sourceRootPath.appending(component: "\(project.name).xcodeproj")
 
+        // Derived files
+        let deleteOldDerivedFiles = try derivedFileGenerator.generate(project: project, sourceRootPath: sourceRootPath)
+
```

Comment:
I wonder if we can do something such that past this point the `InfoPlist` path is set on each target and the remaining components wouldn't need to need to handle all the InfoPlist cases (esp. if that expands later to include more options like `default`). 

I can't think of anything neat to suggest - something for us to consider for next time (we'll no doubt be leveraging the derived generator in more features)

Reply:
Actually this is a good idea. I think we could adjust the generation flow to follow these steps:
- Read the manifests.
- Generate the derived files.
- **Update the manifest to include the derived files.**
- Generate the project.

I really like the idea. It's exactly what we are doing with the manifest project, including it as part of the graph so that the generator generates it.

What do you think if I tackle this in a follow-up PR?

Reply:
Sure 👍